portal/models.py

```diff
@@ -91,7 +82,7 @@ class MediaItem(models.Model):
     store your files '''
     title = models.CharField(_(u"Title"),max_length=200)
     slug = AutoSlugField(populate_from='title',unique=True,verbose_name=_(u"Slug"),help_text=_(u"Slugs are parts of an URL that you can define"))
-    date = models.DateField(_(u"Date"),help_text=_(u"Upload or record date"),default=lambda:localize_input(date.today()))
+    date = models.DateField(_(u"Date"),help_text=_(u"Upload or record date"),null=True)
```

Comment:
You're right. There should be a default. I removed it, because lambda threw an error. 
So the fix would be, @dragetd, to add a function called `localized_today` above the class MediaItem: 

```
def localized_today():
	return [localize_input(date.today())]
``

And then change the marked `null=True` to  `default=localized_today`.

Reply:
You're right. There should be a default. I removed it, because lambda threw an error: [ValueError: Cannot serialize function: lambda while using makemigrations](https://stackoverflow.com/questions/49642982/valueerror-cannot-serialize-function-lambda-while-using-makemigrations)

I've tried some things, but all the solutions had it's downsides.